gptel-openai-extras.el

```diff
@@ -19,7 +19,8 @@
 
```

Comment:
Add your name to the Author field above:
```elisp
;; Authors: Karthik Chikmagalur <...> and pirminj <...>
```
(However you would like to sign your name)